aten/src/ATen/native/cudnn/BatchNorm.cpp

```diff
@@ -146,7 +155,12 @@ std::tuple<Tensor, Tensor, Tensor> cudnn_batch_norm_backward(
   CheckedFrom c = "cudnn_batch_norm_backward";
   setCuDNNStreamToCurrent();
 
-  checkAllDefined(c, {input, grad_output, weight, save_mean, save_var});
+  checkAllDefined(c, {input, grad_output, weight});
+  if (training) {
+    checkAllDefined(c, {save_mean, save_var});
+  } else {
+    throw std::runtime_error("cudnn_batch_norm_backward: !training backwards doesn't seem to work");
```

Comment:
The error message is a bit awkward. Since cuDNN doesn't support `training=false`, why take it as a parameter?

Reply:
I inherited it from the old code, but I see now it makes sense to expunge it entirely.